Tools/MiniBrowserSwiftUI/Shared/BrowserApp.swift

```diff
@@ -28,6 +28,15 @@ import SwiftUI
 @main
 struct BrowserApp: App {
     var body: some Scene {
-        BrowserScene()
+        BrowserScene().commands {
+            CommandGroup(before: CommandGroupPlacement.newItem, addition: {
+                Button(action: {
+                    let currentWindow = NSApp.keyWindow
```

Comment:
Hmm, I'm not sure if this exactly works with the SwiftUI lifecycle stuff. Generally you'd want to use [`@Environment(\.openWindow)`](https://developer.apple.com/documentation/swiftui/environmentvalues/openwindow) to add a new window (and app tabs are technically just windows), but I'm not sure if SwiftUI has support for forcing tabbing to occur.